bundles/core/src/main/java/com/adobe/cq/wcm/core/components/internal/models/v1/ImageImpl.java

```diff
@@ -97,6 +101,10 @@ public class ImageImpl extends AbstractComponentImpl implements Image {
     @Source("osgi-services")
     protected MimeTypeService mimeTypeService;
 
+    @OSGiService(injectionStrategy = InjectionStrategy.OPTIONAL)
+    protected AssetDelivery assetDeliveryService;
```

Comment:
I'd rename all occurrences of `assetDeliveryService` to `assetDelivery` to make reading the code easier.